### PR TITLE
[MB-3502] Orders Information table loads/formats all information on Move Details page

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

This PR adds/modifies the department indicator, orders type, orders type details, tac, and sac fields from the Move Order to the GHC api response to fill the orders table.

The Orders Info table formatting needed some adjustment for dates and human readable names for the different enum fields.  There were also some styling tweaks that may be useful to add to the other tables for Allowances and Customer Info when we touch those next.

## Reviewer Notes

I borrowed most of the api specs from the internal.yaml file.  I did remove the orders types (GHC and NTS) that didn't get cleaned up. This was referenced previously here https://ustcdp3.slack.com/archives/CP6PTUPQF/p1596067160289800

**Storybook Components Affected**
[Orders Table](http://localhost:6006/?path=/story/too-tio-components-orderstable--basic)

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

1. Login to the office app as the TOO
2. Select a move from the queue and scroll to view the Orders table on the Move Details page

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3502) for this change
* [Abstract](https://share.goabstract.com/7b0ee857-f97a-465d-9243-5005f91e9ea4?collectionLayerId=b2b6c7bb-2be5-48d5-b102-9dbff3d4fb03&mode=design)

## Screenshots

Screen Shot 2020-09-03 at 10 44 51 AM

### CODE REVIEW

**Review by brooksr4 — APPROVED** [review]

File: src/services/ghcApi.js [hunk 1]
```diff
@@ -59,6 +59,10 @@ export async function getMTOServiceItems(key, moveTaskOrderID) {
   return makeGHCRequest('mtoServiceItem.listMTOServiceItems', { moveTaskOrderID }, { schemaKey: 'mtoServiceItems' });
 }
 
+export async function getCustomer(key, customerID) {
```
Comment (suzubara) [inline src/services/ghcApi.js:62]: this doesn't seem to be used anywhere? I mean it's still a good thing to add 🤷‍♀️

**Review by suzubara — APPROVED** [review]
looks good to me! nice work continuing to add to constants/formatters/services :D

File: src/services/ghcApi.js (hunk 1 above)
Comment (duncan-truss) [reply src/services/ghcApi.js:62]: Ahh yes I had ambitions of porting over to react-query and then backtracked as i saw there was enough work here already without converting to a functional component.  I know there is probably a chore in the backlog for that work anyway.

**Review by donaldthai — APPROVED** [review]
LGTM :shipit:

**Review by Ryan-Koch — APPROVED** [review]

**Pull request merged**